SoftUni-Discord-Community/skill_forge
# Implement email functionality for skill forge users

Refer to #102 and #110 for more details:

* Email server and email relay service implemented
* Login & Register Forms major improvements and optimization
* Frontend form for user reset password. The user will receive token which is being used to reset his password
* Welcome email template
* Reset password template


Before merging, the following manual tests must be completed:

* Go to the login/register page
* Create new account with a real email address
* Check if you received a welcome email message
* IF yes, is the email message in your Inbox folder and not in Spam/Junk?

* Use the `Forgot Password` form to initiate reset password process
* Follow the steps in the form
* Check if a email address with the generated token is received on your email
* Check if all flash messages works as expected. Try to submit invalid password(without capital letter, withour digits or special character, password does not match, etc) and try to submit the form with missing or invalid token. 
* Submit the form with the correct credentials. Check if the password is was updated and you can login with your new credentials
